src/sass/vic-v2.scss

```diff
@@ -99,6 +99,11 @@ input[type=range] {
   font-size: 1.07em !important;
 }
 
+.id-card-preview {
+  width: 500px;
+  height: 350px;
```

Comment:
IE 10/11 require a height and width to be set on an svg or its container. Percentages can be used but the image gets scaled to the larger dimension and an svg with a fixed aspect ratio does something like this. 
<img width="756" alt="screen shot 2018-02-16 at 10 26 52" src="https://user-images.githubusercontent.com/4998130/36317895-eeba0e34-1303-11e8-99ae-f74a64842e57.png">